initfiles/componentfiles/configschema/xsd/esp_service_smc.xsd

```diff
@@ -121,9 +121,12 @@
                                                          service="ws_topology"/>
                                     <AuthenticateFeature description="Access to own workunits" path="OwnWorkunitsAccess"
                                                          resource="OwnWorkunitsAccess" service="ws_workunits"/>
-                                    <AuthenticateFeature description="Access to others&apos; workunits"
```

Comment:
Actually I should have looked before you squashed.  Here the apostrophe was correct.  It indicates that they own the workunits.

She has access to the others' houses.
She is allowed to deploy resources.

Reply:
My mistake. The apostrophe is needed for others, not for deploy.